src/gui/plugins/view_angle/ViewAngle.hh

```diff
@@ -54,6 +54,12 @@ namespace gazebo
       NOTIFY CamClipDistChanged
     )
 
+    Q_PROPERTY(
+      double camHFOV
+      READ GetCamHFOV
```

Comment:
According to our [style guide](https://gazebosim.org/docs/all/contributing#style-guides), we should avoid accessors that start with `Get`:

```suggestion
      READ CamHFOV
```

Reply:
Changed in f7592a3